src/ProjectService.Broker/Requests/ImageService.cs

```diff
@@ -0,0 +1,109 @@
+    private readonly IRequestClient<ICreateImagesRequest> _rcCreateImages;
+    private readonly IRequestClient<IRemoveImagesRequest> _rcRemoveImages;
+    private readonly IImageInfoMapper _mapper;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+
+    public ImageService(
+      ILogger<ImageService> logger,
+      IRequestClient<IGetImagesRequest> rcGetImages,
+      IRequestClient<ICreateImagesRequest> rcCreateImages,
+      IRequestClient<IRemoveImagesRequest> rcRemoveImages,
+      IImageInfoMapper mapper,
+      IHttpContextAccessor httpContextAccessor)
+    {
+      _logger = logger;
+      _rcCreateImages = rcCreateImages;
+      _rcGetImages = rcGetImages;
+      _rcRemoveImages = rcRemoveImages;
+      _mapper = mapper;
+      _httpContextAccessor = httpContextAccessor;
+    }
+
+    public async Task<List<ImageInfo>> GetImagesAsync(List<Guid> imagesIds, ImageSource imageSource, List<string> errors)
+    {
+      if (imagesIds is null || !imagesIds.Any())
+      {
+        return null;
+      }
+
+      if (imageSource == ImageSource.User)
+      {
+        return (await RequestHandler.ProcessRequest<IGetImagesRequest, IGetImagesResponse>(
+            _rcGetImages,
+            IGetImagesRequest.CreateObj(imagesIds, ImageSource.User),
+            errors,
+            _logger))
+          ?.ImagesData
+          .Select(_mapper.Map).ToList();
+      }
+      
+      if (imageSource == ImageSource.Project)
+      {
+        return (await RequestHandler.ProcessRequest<IGetImagesRequest, IGetImagesResponse>(
+            _rcGetImages,
+            IGetImagesRequest.CreateObj(imagesIds, ImageSource.Project),
+            errors,
+            _logger))
+          ?.ImagesData
+          .Select(_mapper.Map).ToList();
+      }
+
+      return null;
+    }
+
+    public async Task<List<Guid>> CreateImageAsync(List<ImageContent> projectImages, List<string> errors)
+    {
+      return projectImages is null || !projectImages.Any()
+        ? null
+        : (await RequestHandler
+          .ProcessRequest<ICreateImagesRequest, ICreateImagesResponse>(
+            _rcCreateImages,
+            ICreateImagesRequest.CreateObj(
+              projectImages.Select(x => new CreateImageData(
+                x.Name,
+                x.Content,
+                x.Extension,
+                _httpContextAccessor.HttpContext.GetUserId()))
+              .ToList(),
+              ImageSource.Project),
+            errors,
+            _logger)).ImagesIds;
+    }
+
+    public async Task<bool> RemoveImages(List<Guid> imagesIds, List<string> errors)
+    {
+      return imagesIds is null || imagesIds.Any()
+        ? false
+        : (await RequestHandler
+          .ProcessRequest<IRemoveImagesRequest, bool>(
+            _rcRemoveImages,
+            IRemoveImagesRequest.CreateObj(imagesIds: imagesIds, imageSource: ImageSource.User)),
+            errors,
+            _logger).Item1;
```

Comment:
exception is something wrong, _ogger)?.Body

Reply:
done